core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -721,11 +721,34 @@ class AutoTuner(
     recommendFileCache()
     recommendMaxPartitionBytes()
     recommendShufflePartitions()
+    recommendKyroSerializerSetting()
     recommendGCProperty()
     recommendClassPathEntries()
     recommendSystemProperties()
   }
 
+  // if the user set the serializer to use Kryo, make sure we recommend using the GPU version
+  // of it.
+  def recommendKyroSerializerSetting(): Unit = {
+      getPropertyValue("spark.serializer") match {
+        case Some(f) if f.contains("org.apache.spark.serializer.KryoSerializer") =>
+          val registratorToUse = if (getPropertyValue("spark.kryo.registrator").isDefined) {
+            // need to append our GpuKryoRegistrator to existing ones
+            val existingRegistrars = getPropertyValue("spark.kryo.registrator")
+            if (!existingRegistrars.get.isEmpty) {
+              "com.nvidia.spark.rapids.GpuKryoRegistrator" + "," + existingRegistrars.get
```

Comment:
nit: Reading this, I assume that the configuration value is a comma-separated string. Spark picks the registrars in order. If that's correct, then we can fix the comment in line 736 to be more accurate (whereas appending will be last in the list).